internal/payloads/account_identity.go

```diff
@@ -0,0 +1,30 @@
+package payloads
+
+import (
+	"net/http"
+
+	"github.com/RHEnVision/provisioning-backend/internal/clients"
+	"github.com/go-chi/render"
+)
+
+type AccountIdentityResponse struct {
+	*clients.AccountIdentity
+}
+
+func NewAccountIdentityResponse(awsAccountId string) render.Renderer {
+	return &AccountIdentityResponse{
+		&clients.AccountIdentity{
+			AWSDetails: &clients.AccountDetailsAWS{
+				AccountID: awsAccountId,
+			},
+		},
+	}
+}
+
+func (s *AccountIdentityResponse) Bind(_ *http.Request) error {
+	return nil
+}
```

Comment:
You only need Render for responses, Bind is only needed for Requests. So this method can be deleted I think.